specification/nhais-adaptor.yaml

```diff
@@ -170,20 +196,26 @@ paths:
 
         https://digital.nhs.uk/services/nhais/guide-to-nhais-gp-links-documentation
 
-        Three fields are deprecated and not supported by the adaptor:
+        Three fields are DEPRECATED and not supported by the adaptor:
 
         * RPP Mileage
         * Blocked Route/Special District Marker
         * Walking Units
 
+        REQUIRED - If any of these data items are not provided or are blank then the adaptor will produce a 400 error
```

Comment:
*REQUIRED - THE ONLY FIELDS THAT ARE VALIDATED BY THE ADAPTOR. If any of these data items are not provided or are blank then the adaptor will produce a 400 error

Reply:
as discussed in a separate PR we'll add a section to the top-level description about input validation that covers all transaction types